### DISCUSSION TITLE
"View Creation Log" results in error, command not found

### DISCUSSION THREAD

**Discussion by pamelafox** [community/community#52748] [Codespaces]
### Select Topic Area

Bug

### Body

This bug occurred yesterday while I was teaching a workshop with Codespaces.

I opened https://github.com/Azure-Samples/msdocs-flask-postgresql-sample-app in Codespaces in the browser.

Once open, I used the command palette to find "Codespaces: View Creation Log" command.
I clicked on the command and saw error:

![Screenshot that says command github.codespaces.viewCreationLog not found](https://user-images.githubusercontent.com/297042/231596507-9d942408-24fa-409c-82e7-9422d4f5d1a0.png)

Error says: "command 'github.codespaces.viewCreationLog' not found"

I was able to view the creation log later that day in other Codespaces, so it may have been an intermittent error or an error that's since been fixed.

**Comment by osortega** [comment]
Hi @pamelafox! 

That is definitely not expected, thanks for reporting this.
I just filed an internal issue to fix it.
